src/main/scala/com/comcast/xfinity/sirius/uberstore/segmented/SegmentedCompactor.scala

```diff
@@ -165,6 +206,7 @@ private [segmented] class SegmentedCompactor(maxDeleteAgeMillis: Long) {
     left.foreach(target.writeEntry)
     right.foreach(target.writeEntry)
     target.setApplied(applied = left.isApplied && right.isApplied)
+    target.setInternallyCompacted(compacted = false)
```

Comment:
Yeah, there is that one case where it could be true. I mostly didn't want to overcomplicate things, but it's reasonable to be as precise as possible here. I'll switch it up.